mungegithub/mungers/block_paths.go

```diff
@@ -26,11 +26,18 @@ import (
 	"k8s.io/kubernetes/pkg/util/yaml"
 
 	"github.com/golang/glog"
+	github_api "github.com/google/go-github/github"
 	"github.com/spf13/cobra"
 )
 
+const (
+	blockPathFormat = `Adding label:%s because PR changes docs prohibited to auto merge
```

Comment:
I'm not a native speaker, but I don't really understand this comment. Is something along the lines "You need to update docs"?